src/angular-signature-pad.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+const angular = require('angular')
+const SignaturePad = require('signature_pad')
+
+BmSignaturePadController.$inject = ['$scope', '$element', '$window', '$log']
+
+function BmSignaturePadController ($scope, $element, $window, $log) {
+  if (!SignaturePad) {
+    $log.error('SignaturePad is required')
+    return
+  }
+  const vm = this
```

Comment:
because you are using fat arrows for the functions, the value of `this` _should_ be the same as `this` in the closure scope, i.e. a `BmSignaturePadController`  instance. Which means using `const vm = this` shouldn't be necessary. or is it, because: angular ?